src/components/Navbar/SettingsDrawer/TranslationSelectionBody.tsx

```diff
@@ -172,4 +174,5 @@ const TranslationSelectionBody = () => {
   );
 };
 
+// eslint-disable-next-line max-lines
```

Comment:
Instead of doing that, we should split the file into smaller components.

Reply:
What additional component should I create and where? Just confirming if there's any specific pattern that you're following?